src/config/dynamic-theme-fixes.config

```diff
@@ -4727,6 +4727,19 @@ img[src*=".png"]
 
 ================================
 
+yandex.*/maps
+
+INVERT
+.content-panel-header__logo
+.whats-here-preview__control-search-icon
+.close-button._color_black._circle
+.business-social-links-view__icon
+.social-share-view_discovery-small
+.social-share-view_discovery-large
+.orgpage-social-links-view__icon
+
```

Comment:
add `.map-container > ymaps > ymaps > ymaps > ymaps`
So it will look likes
![](https://i.imgur.com/a6mlUtB.png)